about.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="about.css">
+  <title>Capstone Project x01</title>
+</head>
+<body>
+        <div class="container">
+          <div class="loggings desktop">
+            <a href="#"><img src="./assets/desktop/facebook.png" alt="facebook"></a>
+            <a href="#"><img src="./assets/desktop/twitter.png" alt="twitter"></a>
+            <a href="#"><p>Logout</p></a>
+            <a href="#"><p>Mypage</p></a>
+            <a href="#"><p>Korean</p></a>
+          </div>
+          <header>
+            <img class="hambuger" src="./assets/mobile//hamburger.png" alt="harmbuger icon"/>
+            <nav class="navigator toggle-ON">
+              <button class="exit-btn">X</button>
+              <a href="index.html"><img class="desktop" src="./assets/mobile/logo.svg" alt="logo"></a>
+              <ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="#">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li class="desktop pdc"><a href="#">PD Campaign</a></li>
+              </ul>
+              <div>
+                <hr>
+              </div>
+            </nav>
+          </header>
+          <main>
+            <div>
+            <h4 class="sharing">
+              "Hello! Sharing world"
+            </h4>
+            <h1>
+              POLISH YOUR SKILLS AND<span class="title-span"><br>GET READY</span>
+            </h1>
+            <hr>
+            </div>
+            <p class="support">            
+              The Global podcasting submit brings together the community of experts,
+              academics, activists who comprise the Creative commons affiliates network,
+              in a different country every two. This year, we hope to expand out invitation
+              list including organisations and individuals.<br><span class="desktop">So if your are active and engaged
+              in the worlds of open content and knowledge - Free software advocates, Wikipedias, open knowledge, galleries,
+              libraries, museums, archives, government and foundations - We hope you will conside joining us this year to build
+              a stronger more vibrant community.
+              </span>
+            </p>
+            <p>
+              Please contact us per Email for any further questions about PD Global Submit 2020!
+              <br><span class="mail"><a href="#">podcasting@pod.org</a></span>
+            </p>
+          </main>
+        </div>
```

Comment:
- [ ] `<main>` tag is expected to be the parent of all other tags in your file, not as a child to any tag. Kindly look into that and fix it. You can get more info about that [here](https://www.youtube.com/watch?v=lVmX-7XEyd4).